harness/src/lib.rs

```diff
@@ -625,19 +632,22 @@ impl Mollusk {
                 EnvironmentConfig::new(
                     Hash::default(),
                     /* blockhash_lamports_per_signature */ 5000, // The default value
-                    0,
-                    &|_| 0,
-                    Arc::new(self.feature_set.clone()),
+                    &MockInvokeContextCallback {},
+                    &self.feature_set.runtime_features(),
                     &sysvar_cache,
                 ),
                 self.logger.clone(),
-                self.compute_budget,
+                SVMTransactionExecutionBudget::default(),
+                Default::default(),
             );
             if let Some(precompile) = get_precompile(&instruction.program_id, |feature_id| {
-                invoke_context.get_feature_set().is_active(feature_id)
+                // invoke_context.get_feature_set().is_active(feature_id)
+                // There is not way to take a SVMFeatureSet and see if the feature pubkey
+                // matches a field
```

Comment:
@buffalojoec maybe you have some ideas here. Sadly, we don't have a way to take a feature pubkey and see if it is enabled within a `SVMFeatureSet`.